ui/shared/components/panel/search/VariantSearchFormContainer.jsx

```diff
@@ -19,8 +19,17 @@ VariantSearchFormContainer.propTypes = {
 }
 
 const mapDispatchToProps = (dispatch, ownProps) => ({
-  onSubmit: (searchParams) => {
-    dispatch(navigateSavedHashedSearch(searchParams, ownProps.history.push, ownProps.resultsPath))
+  onSubmit: ({ search, ...searchParams }) => {
+    const { rawItems } = search?.locus || ''
+    const formattedRawItems = (rawItems && typeof rawItems === 'object') ? toUniqueCsvString(Object.values(rawItems)) : rawItems
```

Comment:
probably makes more sense to move these lines inside the `if` block

Reply:
Done.